tests/services/data.service.spec.ts

```diff
@@ -0,0 +1,149 @@
+import { GithubService } from '../../src/app/core/services/github.service';
+import { DataService } from '../../src/app/core/services/data.service';
+import { autoSpy } from '../auto-spy';
+import { of } from 'rxjs';
+
+const csvString = `
+role,name,team

```

Comment:
Let's place this in a constant, as it may be useful for testing UserService

Reply:
Moved some possible shared constants into a separate file